### PR TITLE
Feature/static content

### PR DESCRIPTION
**Pull request by test0n3**
# Content:
- Welcome
- Role
- Walkthrough

# Header
- Menu functionality
- Menu content

### CODE REVIEW

File: simcase-game/src/App.js [hunk 1]
```diff
@@ -1,11 +1,22 @@
...
+import Header from "./components/Header";
 
 function App() {
   return (
-    <div>
-      <GameUI />
-    </div>
+    <>
+      <Global
+        styles={{ body: { margin: 0, padding: 0, fontFamily: "sans-serif" } }}
```
Comment (CesarCachay) [inline simcase-game/src/App.js:13]: Could you explain me please what is the purpose of the Global component?

File: simcase-game/src/components/Header.js [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+            <button
+              onClick={() => setState({ showDialog: true })}
+              css={menuButton}
+            >
+              &#9776;
+            </button>
+
+            <Dialog
+              isOpen={state.showDialog}
+              onDismiss={() => setState({ showDialog: true })}
```
Comment (CesarCachay) [inline simcase-game/src/components/Header.js:84]: Why do you thing separate this functions outside the return?

File: simcase-game/src/components/Walkthrough.js [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+import { jsx } from "@emotion/core";
+
+function Walkthrough() {
+  const contentStyle = {
+    padding: 10
+  };
+  return (
+    <main css={contentStyle}>
+      <h2>Walkthrough</h2>
+      <img src="assets/img/walkthrough.gif" alt="Walkthrough to use the app" />
```
Comment (CesarCachay) [inline simcase-game/src/components/Walkthrough.js:12]: Could you explain me which is the purpose of the attribute "alt" please?

**Review by CesarCachay — APPROVED** [review]

File: simcase-game/src/components/Header.js [hunk 4]
```diff
@@ -0,0 +1,115 @@
...
+              </ul>
+            </Dialog>
+          </div>
+        )}
+      </Component>
+    </header>
+  );
+}
+
+export default Header;
```
Comment (dotconde) [inline simcase-game/src/components/Header.js:115]: Well done, but I think you might try to separate your component into more components ('divide and conquer')

File: simcase-game/yarn.lock [hunk 5]
```diff
@@ -1205,6 +1205,29 @@
   resolved "https://registry.yarnpkg.com/@nodelib/fs.stat/-/fs.stat-1.1.3.tgz#2b5a3ab3f918cca48a8c754c08168e3f03eba61b"
```
Comment (dotconde) [inline simcase-game/yarn.lock:1205]: You shouldn't upload yarn.lock within your PR

**Review by dotconde — APPROVED** [review]

**Review by dotconde — APPROVED** [review]

**Comment by KattyaCuevas** [comment]
Issues #4 #5 #1

**Pull request merged**